tests/test_fmi3_sim.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2025 Modelon AB
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import re
+import logging
+from io import StringIO
+from pathlib import Path
+
+import pytest
+import numpy as np
+
+from pyfmi import load_fmu
+
+this_dir = Path(__file__).parent.absolute()
+FMI3_REF_FMU_PATH = Path(this_dir) / 'files' / 'reference_fmus' / '3.0'
+
+class TestSimulation:
+    """Tests involving simulation of FMUs for FMI 3."""
+
+    def test_simulate(self):
+        """Test simulate VDP model and verify the integrity of the results. """
+        fmu = load_fmu(FMI3_REF_FMU_PATH / "VanDerPol.fmu")
+        results = fmu.simulate()
+
+        assert results['x0'][0] == 2.0
+        assert results['x1'][0] == 0.0
+        assert results['x0'][-1] == pytest.approx( 2.00814337)
+        assert results['x1'][-1] == pytest.approx(-0.04277047)
+        np.testing.assert_equal(results['mu'], np.ones(501))
```

Comment:
Could we change this one as to not regression test the default NCP?

Reply:
Good idea, fixed